content/bn/docs/concepts/overview/what-is-kubernetes.md

```diff
@@ -0,0 +1,94 @@
+
+## কুবারনেটিস কি না
+
+কুবারনেটিস একটি ঐতিহ্যগত, সর্ব-অন্তর্ভুক্ত PaaS (পরিষেবা হিসাবে প্ল্যাটফর্ম) সিস্টেম নয়। যেহেতু Kubernetes হার্ডওয়্যার স্তরের পরিবর্তে কন্টেইনার স্তরে কাজ করে, তাই এটি PaaS অফারগুলির জন্য সাধারণভাবে কিছু প্রযোজ্য বৈশিষ্ট্য প্রদান করে, যেমন স্থাপনা, স্কেলিং, লোড ব্যালেন্সিং, এবং ব্যবহারকারীদের তাদের লগিং, পর্যবেক্ষণ এবং সতর্কতা সমাধানগুলিকে একীভূত করতে দেয়৷ যাইহোক, কুবারনেটিস একচেটিয়া নয়, এবং এই ডিফল্ট সমাধান ঐচ্ছিক এবং প্লাগযোগ্য। কুবারনেটিস বিকাশকারী প্ল্যাটফর্ম তৈরির জন্য বিল্ডিং ব্লক সরবরাহ করে, কিন্তু যেখানে এটি গুরুত্বপূর্ণ সেখানে ব্যবহারকারীর পছন্দ এবং নমনীয়তা সংরক্ষণ করে।
+
+কুবারনেটিস:
+
+* সমর্থিত অ্যাপ্লিকেশনের ধরন সীমাবদ্ধ করে না। কুবারনেটিস-এর লক্ষ্য স্টেটলেস, স্টেটফুল, এবং ডেটা-প্রসেসিং ওয়ার্কলোড সহ অত্যন্ত বৈচিত্র্যময় কাজের লোড সহায়তা করা। যদি একটি অ্যাপ্লিকেশন একটি পাত্রে চলতে পারে তবে এটি কুবারনেটে দুর্দান্ত চালানো সম্ভব।
+* সোর্স কোড স্থাপন করে না এবং আপনার অ্যাপ্লিকেশন তৈরি করে না। ক্রমাগত ইন্টিগ্রেশন, ডেলিভারি, এবং ডিপ্লয়মেন্ট (CI/CD) ওয়ার্কফ্লোগুলি সংস্থার প্রকৃতির উপর এবং পছন্দগুলির পাশাপাশি প্রযুক্তিগত প্রয়োজনীয়তা দ্বারা নির্ধারিত হয়। 
+* মিডলওয়্যার (উদাহরণস্বরূপ, বার্তা বাস), ডেটা-প্রসেসিং ফ্রেমওয়ার্ক (উদাহরণস্বরূপ, স্পার্ক), ডাটাবেস (উদাহরণস্বরূপ, মাইএসকিউএল), ক্যাশে, বা ক্লাস্টার স্টোরেজ সিস্টেম (উদাহরণস্বরূপ, Ceph) এর মতো অ্যাপ্লিকেশন-স্তরের পরিষেবা প্রদান করে না। অন্তর্নির্মিত পরিষেবা হিসাবে। এই ধরনের উপাদানগুলি কুবারনেটিস-এ চলতে পারে, এবং/অথবা পোর্টেবল মেকানিজম, যেমন [ওপেন সার্ভিস ব্রোকার](https://openservicebrokerapi.org/) এর মাধ্যমে Kubernetes-এ চলমান অ্যাপ্লিকেশনের মাধ্যমে অ্যাক্সেস করা যেতে পারে।
+* লগিং, পর্যবেক্ষণ, বা সতর্কতা সমাধান নির্দেশ করে না। এটি ধারণার প্রমাণ হিসাবে কিছু ইন্টিগ্রেশন এবং মেট্রিক্স সংগ্রহ ও রপ্তানি করার প্রক্রিয়া প্রদান করে।
+* কনফিগারেশন ভাষা/সিস্টেম প্রদান করে না বা বাধ্যতামূলক করে না (উদাহরণস্বরূপ, জসননেট)। এটি একটি ঘোষণামূলক API প্রদান করে যা ঘোষণামূলক স্পেসিফিকেশনের নির্বিচারে ফর্ম দ্বারা লক্ষ্যধারন করা যেতে পারে।
+* কোন ব্যাপক যান্ত্রিক কনফিগারেশন, রক্ষণাবেক্ষণ, ব্যবস্থাপনা, বা স্ব-নিরাময় সিস্টেম প্রদান বা গ্রহণ করে না।
+*উপরন্তু, কুবারনেটিস একটি নিছক অর্কেস্ট্রেশন সিস্টেম নয়। প্রকৃতপক্ষে, এটি অর্কেস্ট্রেশনের প্রয়োজনীয়তা দূর করে। অর্কেস্ট্রেশনের প্রযুক্তিগত সংজ্ঞা হল একটি সংজ্ঞায়িত কর্মপ্রবাহের সঞ্চালন: প্রথমে A, তারপর B, তারপর C করুন। বিপরীতে, Kubernetes স্বাধীন, সংমিশ্রণযোগ্য নিয়ন্ত্রণ প্রক্রিয়াগুলির একটি সেট নিয়ে গঠিত যা বর্তমান অবস্থাকে প্রদত্ত পছন্দসই অবস্থার দিকে ক্রমাগত চালিত করে। আপনি A থেকে C পর্যন্ত কিভাবে যাবেন তা বিবেচ্য নয়। কেন্দ্রীভূত নিয়ন্ত্রণেরও প্রয়োজন নেই। এর ফলে এমন একটি সিস্টেম তৈরি হয় যা ব্যবহার করা সহজ এবং আরও শক্তিশালী, মজবুত, স্থিতিস্থাপক এবং বর্ধনশীল।
+
+
+
+## {{% heading "এরপর কি" %}}
+
+*   কটাক্ষপাত করুন [Kubernetes Components](/docs/concepts/overview/components/)
```

Comment:
Can you check this sentence? `কটাক্ষপাত করুন [Kubernetes Components]` . The sentence has not been properly localized